examples/crm/src/root/CRM.tsx

```diff
@@ -0,0 +1,158 @@
+import {
+    Admin,
+    CustomRoutes,
+    ListGuesser,
+    RaThemeOptions,
+    Resource,
+    defaultTheme,
+    localStorageStore,
+} from 'react-admin';
+
+import { Route } from 'react-router';
+import Layout from '../Layout';
+import { authProvider } from '../authProvider';
+import companies from '../companies';
+import contacts from '../contacts';
+import { Dashboard } from '../dashboard/Dashboard';
+import { dataProvider } from '../dataProvider';
+import deals from '../deals';
+import { LoginPage } from '../login/LoginPage';
+import { SignupPage } from '../login/SignupPage';
+import { SettingsPage } from '../settings/SettingsPage';
+import {
+    ConfigurationContextValue,
+    ConfigurationProvider,
+} from './ConfigurationContext';
+import {
+    defaultCompanySectors,
+    defaultContactGender,
+    defaultDealCategories,
+    defaultDealPipelineStatuses,
+    defaultDealStages,
+    defaultLogo,
+    defaultNoteStatuses,
+    defaultTaskTypes,
+    defaultTitle,
+} from './defaultConfiguration';
+
+// Define the interface for the CRM component props
+interface CRMProps {
```

Comment:
This can be simplified to : 

```ts
type CRMProps = {
    lightTheme?: RaThemeOptions;
    darkTheme?: RaThemeOptions;
} & Partial<ConfigurationContextValue>;
```